frameworks/hdfs/src/main/dist/svc.yml

```diff
@@ -93,17 +93,23 @@ pods:
         readiness-check:
           cmd: >
```

Comment:
You could always switch this to `|`, and then do a `set -e` at the top of the command, then you can skip a lot of &&-ing.

Reply:
Done here and throughout the other `cmd`s.